### DISCUSSION TITLE
Solo2 doesn't work on Firefox but works on Chrome

### DISCUSSION THREAD

**Discussion by kriive** [solokeys/solo2#151] [Q&A]
Hey! I received a solo2 and updated to the latest firmware.
Somehow, in Firefox 109 (and all FF version before), the solo2 doesn't get recognized, while on Google Chrome it does. Why is that? Anyone has a clue?

I am using Arch Linux.

**Comment by msiqx9** [comment]
I have had the Solo v2 for three months.  It has the latest firmware.

Following your post, I just tested my Solo v2 key on my Ubuntu laptop (22.10) with Firefox 108.  I was able to log into 5 different sites using the Solo v2.  On the last site (Bitwarden) I de-registered my Solo v2 key and re-registered it (WebAuthn) and that also went OK.   The Solo v2 key also works fine for me in Windows 11 22H2 with Firefox 109.  Maybe there is an issue in Arch Linux?

**Reply by kriive** [reply]
I think so, I am beginning to suspect that Arch is to blame
